.github/actions/pr-manager/index.js

```diff
@@ -0,0 +1,230 @@
+const core = require('@actions/core');
+const { context, getOctokit } = require('@actions/github');
+
+const token = core.getInput('github_token', { required: true });
+const octokit = getOctokit(token);
+const github = octokit.rest;
+
+// Regex that matches any snapshots, including -pre and -rc version numbers.
+const snapshotRegex = new RegExp("\d\dw\d\d[a-e]|\d\.\d\d\.\d\d?-(?:pre|rc)\d")
+
+// Any labels that are used in the code.
+const fcpLabel = 'final-comment-period';
+const tinyLabel = "s: tiny";
+const smallLabel = "s: small";
+const medLabel = "s: medium";
+const largeLabel = "s: large";
+const newLabel = "T: new";
+const refactorLabel = "T: refactor";
+const docLabel = "T: documentation";
+const toolLabel = "T: toolchain";
+
+// If someone is added to the Mappings team, add their Github username here.
+const mappingsTeam = [
```

Comment:
since Quilt teams are granular, it should be possible to get the mappings team from the API instead of manually maintaining it

Reply:
Updated this!